scripts/generators/foundry.py

```diff
@@ -1,20 +1,21 @@
-import yaml
 from glob import glob
 import json
 import os
-import pandas as pd
-import re
 from shutil import copy
 import fire
-import time
 import datetime
 
-secrets = json.load(open('../../secrets.json'))
+# Moves with fields that are problematic for the export
+# - Non-moves such as "Run Away" are excluded
+# - Struggle and Growl have attributes like "Strength/special"
```

Comment:
Those are in fact, interchangable. Not sure if Foundry can support the either other, but the bot and srd do via having two entries, one with each stat.

Reply:
Yup, the split versions like "Growl (Cute)" etc. are included as separate entries, only the aggregate one is skipped

Reply:
Cool, Didn't realize that from the note. 👍